specification/gedcom-3-structures-1-organization.md

```diff
@@ -1188,6 +1188,7 @@ This specification does not support places where a region name contains a comma.
 n SOUR @<XREF:SOUR>@                       {1:1}  g7:SOUR
   +1 RESN <List:Enum>                      {0:1}  g7:RESN
   +1 PAGE <Text>                           {0:1}  g7:PAGE
+     +2 <<SOURCE_CITATION>>                {0:M}
```

Comment:
Luther argues to promote this up a level, and also to add `SOURCE_CITATION` under the source record as well.

Reply:
Done